source/accessibility.html.md.erb

```diff
@@ -45,18 +43,8 @@ The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
 
 DfE is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
 
-## Compliance status
```

Comment:
Compliance statement needs to stay - https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website

Reply:
Thanks, I've added it back